contracts/tokens/MetaStudioToken.sol

```diff
@@ -190,6 +202,69 @@ contract MetaStudioToken is
     super._spendAllowance(_owner, spender, amount);
   }
 
+  function granularity()
+    public
+    view
+    override(ERC777Upgradeable)
+    returns (uint256)
+  {
+    return super.granularity();
+  }
+
+  function send(
+    address recipient,
+    uint256 amount,
+    bytes memory data
+  ) public override(ERC777Upgradeable) {
+    super.send(recipient, amount, data);
+  }
+
+  function authorizeOperator(address operator)
+    public
+    override(ERC777Upgradeable)
+  {
+    super.authorizeOperator(operator);
+  }
+
+  function burn(uint256 amount, bytes memory data)
+    public
+    override(ERC777Upgradeable)
+  {
+    super.burn(amount, data);
+  }
+
+  function revokeOperator(address operator) public override(ERC777Upgradeable) {
+    super.revokeOperator(operator);
+  }
+
+  function defaultOperators()
+    public
+    view
+    override(ERC777Upgradeable)
+    returns (address[] memory)
+  {
+    return super.defaultOperators();
+  }
+
+  function operatorSend(
+    address sender,
+    address recipient,
+    uint256 amount,
+    bytes memory data,
+    bytes memory operatorData
+  ) public override(ERC777Upgradeable) {
+    super.operatorSend(sender, recipient, amount, data, operatorData);
+  }
+
+  function operatorBurn(
+    address account,
+    uint256 amount,
+    bytes memory data,
+    bytes memory operatorData
+  ) public override(ERC777Upgradeable) {
+    super.operatorBurn(account, amount, data, operatorData);
+  }
+
```

Comment:
Why do we need to override those methods if there's only one class implementing them (ERC777Upgradeable) ?